adal/src/androidTest/java/com/microsoft/aad/adal/AcquireTokenSilentHandlerTest.java

```diff
@@ -45,93 +45,94 @@
 import android.test.suitebuilder.annotation.SmallTest;
 
 /**
- * Tests for verifying acquire token silent flow. 
+ * Tests for verifying acquire token silent flow.
  */
-public final class AcquireTokenSilentHandlerTest extends AndroidTestCase{
+public final class AcquireTokenSilentHandlerTest extends AndroidTestCase {
     /**
      * Check case-insensitive lookup
      */
     private static final String VALID_AUTHORITY = "https://Login.windows.net/Omercantest.Onmicrosoft.com";
 
-    protected final static int CONTEXT_REQUEST_TIME_OUT = 20000;
+    protected static final int CONTEXT_REQUEST_TIME_OUT = 20000;
 
-    protected final static int ACTIVITY_TIME_OUT = 1000;
+    protected static final int ACTIVITY_TIME_OUT = 1000;
 
-    static final String testClientId = "650a6609-5463-4bc4-b7c6-19df7990a8bc";
-
-    static final String testResource = "https://omercantest.onmicrosoft.com/spacemonkey";
-    
     static final String TEST_IDTOKEN = "eyJ0eXAiOiJKV1QiLCJhbGciOiJub25lIn0.eyJhdWQiOiJlNzBiMTE1ZS1hYzBhLTQ4MjMtODVkYS04ZjRiN2I0ZjAwZTYiLCJpc3MiOiJodHRwczovL3N0cy53aW5kb3dzLm5ldC8zMGJhYTY2Ni04ZGY4LTQ4ZTctOTdlNi03N2NmZDA5OTU5NjMvIiwibmJmIjoxMzc2NDI4MzEwLCJleHAiOjEzNzY0NTcxMTAsInZlciI6IjEuMCIsInRpZCI6IjMwYmFhNjY2LThkZjgtNDhlNy05N2U2LTc3Y2ZkMDk5NTk2MyIsIm9pZCI6IjRmODU5OTg5LWEyZmYtNDExZS05MDQ4LWMzMjIyNDdhYzYyYyIsInVwbiI6ImFkbWluQGFhbHRlc3RzLm9ubWljcm9zb2Z0LmNvbSIsInVuaXF1ZV9uYW1lIjoiYWRtaW5AYWFsdGVzdHMub25taWNyb3NvZnQuY29tIiwic3ViIjoiVDU0V2hGR1RnbEJMN1VWYWtlODc5UkdhZEVOaUh5LXNjenNYTmFxRF9jNCIsImZhbWlseV9uYW1lIjoiU2VwZWhyaSIsImdpdmVuX25hbWUiOiJBZnNoaW4ifQ.";
-    
+
     static final String TEST_IDTOKEN_USERID = "4f859989-a2ff-411e-9048-c322247ac62c";
-    
+
     static final String TEST_IDTOKEN_UPN = "admin@aaltests.onmicrosoft.com";
-    
+
+    static final int HTTP_OK = 200;
+    static final int HTTP_BAD_REQUEST = 400;
+    static final int HTTP_INTERNAL_SERVER_ERROR = 500;
```

Comment:
we should use HttpUrlConnection.HTTP_OK, HTTP_BAD_REQUEST, HTTP_INTERNAL_ERROR instead.